public/scripts/power-user.js

```diff
@@ -257,6 +258,7 @@ let power_user = {
     },
 
     reasoning: {
+        name: 'DeepSeek R1',
```

Comment:
You want to name the default preset "DeepSeek R1"?
I mean, that's what most use it for, sure. But I think a lot of locally trained models use the same format.

I don't mind, but I am just thinking about months down the line, when we are stuck with a default preset and default name that is named after the first big model that had it.

Reply:
I would prefer DeepSeek personally, than a specific model number. Think that's more future-proof.